tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2129,7 +2129,9 @@ template <typename Container>
 Status ConvertStridedSliceHelper(OpConverterParams* params,
                                  const TRT_TensorOrWeights& input,
                                  Container begin, Container size,
-                                 const Container& stride) {
+                                 const Container& stride,
+                                 bool reshape_after = false,
```

Comment:
Please use const pointer type for `final_shape` and remove `reshape_after`. So if `final_shape` is `nullptr` we don't need to reshape.